src/qibolab/_core/instruments/qblox/sequence/program.py

```diff
@@ -540,7 +540,7 @@ def long_wait(duration: int) -> list[Line]:
     iterations = duration // MAX_WAIT
     remainder = duration % MAX_WAIT
     register = Register(number=n)
-    label = f"wait-{n}"
+    label = f"wait{n}"
```

Comment:
I think q1asm does not like the `-` character because it gives
```py
assembler failed with 2 errors:
line 8 column 7: Syntax error: AsmParser: spurious characters after statement:
wait-60: wait      65535
-------^^^^^^^^^^^^^^^^^^^^^^^^^

line 9 column 24: Syntax error: Invalid address label reference:
         loop      R60,@wait-60
------------------------^^^^^^^
```

Reply:
Ah, that's fine. I had to fix this part anyhow (and still need), but this is certainly among the sort of errors I have in mind when I said that I expected bugs.

Thanks for the fix!